contentcuration/contentcuration/frontend/channelEdit/utils.js

```diff
@@ -178,6 +238,36 @@ export function secondsToHms(seconds) {
   return hms;
 }
 
+/**
+ * Set a default completion model for audiovisual content
+ * equal to the file length in seconds (parsed), or a fallback placeholder
+ *
+ * @param {Object} node
```

Comment:
Non-blocking. `node` attribute documentation seems to be a mismatch as the function is taking files of a node via its parameter